aiida/orm/utils/mixins.py

```diff
@@ -105,13 +107,45 @@ def _set_function_starting_line_number(self, function_starting_line_number):
         """
         self.set_attribute(self.FUNCTION_STARTING_LINE_KEY, function_starting_line_number)
 
+    @property
+    def function_number_of_lines(self):
+        """Return the number of lines of the wrapped function in its source file.
+
+        :returns: the number of lines or None
+        """
+        line_count = self.get_attribute(self.FUNCTION_NUMBER_OF_LINES_KEY)
```

Comment:
I assume `get_attribute` with specifying default return value will raise the exception for old nodes, correct?

Reply:
Yes, you can just pass the default to the `get_attribute` call though
```suggestion
        line_count = self.get_attribute(self.FUNCTION_NUMBER_OF_LINES_KEY, None)
```

Reply:
I set the default value to `-1` so for the old nodes this function will return the content to end of the file.